TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -559,41 +533,40 @@ class RouteOptionsViewController: UIViewController {
         }
     }
 
-    private func processRequest(result: Result<Response<RouteSectionsObject>>, requestURL: String, endPlace: Place) {
-        JSONFileManager.shared.logURL(timestamp: Date(), urlName: "Route requestUrl", url: requestURL)
+    private func processRequest(start: Place, end: Place, time: Date, type: SearchType) {
+        if let result =  getRoutes(start: start, end: end, time: time, type: type) {
+            result.observe(with: { [weak self] result in
+                guard let `self` = self else { return }
```

Comment:
Pretty sure you can drop the tick marks since Swift 4.2.

Reply:
I hear ya! I think we have tick marks throughout the entire codebase right now? Do we wanna try to fix all of it at once?

Reply:
Honestly we could probably do it in one go with a find and replace of `` `self` `` –> `self` in the project.